source/_docs/continuous-integration.md

```diff
@@ -17,18 +17,76 @@ Continuous Integration (CI) is a method of running automated unit and integratio
 
 You can use Terminus for scripting many operations. For example, a post-commit hook can trigger Jenkins to create a Multidev environment with the latest code on master and the content from Live, then run automated browser tests using [Selenium](https://github.com/SeleniumHQ/selenium).
 
-## Drupal SimpleTest
-
-[SimpleTest](https://drupal.org/project/simpletest) is a testing framework based on the [SimpleTest PHP library](https://github.com/simpletest/simpletest) that is included with Drupal core. If you are creating a custom web application, you should consider including SimpleTests of your module functionality.
-
-After enabling the SimpleTest module, you can use Drush to remotely execute SimpleTest on your Pantheon site. For example, if you want to execute the UserSaveTestCase test and generate XML output into a writeable directory, use the following command:
-
-    SITE_NAME=yoursitename
-    ENV=dev
-    drush @pantheon.$SITE_NAME.$ENV test-run -l https://$ENV-$SITE_NAME.pantheon.io/ UserSaveTestCase --xml='sites/default/files'
-
-The end result is written to https://dev-yoursitename.pantheon.io/sites/default/files/UserSaveTestCase.xml

```

Comment:
@ccjjmartin Is this line no longer true? Where are the results?

Reply:
P.S. I'm pretty sure this is the _last_ question I have about this PR.

Reply:
@alexfornuto The way that we run the command, the results are output directly to the terminal (screen) when run.  I haven't tested the command with a file output option `--xml='sites/default/files' (but it is probably still there).  In my opinion it works better seeing the results in the case of an active developer or from CIs (which normally save the logs already for us).  So yeah, we should be good to remove that.